packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -234,29 +235,39 @@ export const inviteMembersWithNoInviterPermissionCheck = async (
 
 const inviteMembers = async ({ ctx, input }: InviteMemberOptions) => {
   const { user: inviter } = ctx;
+  const { usernameOrEmail, role, isPlatform } = input;
 
-  const inviterOrg = inviter.organization;
   const team = await getTeamOrThrow(input.teamId);
+  const requestedSlugForTeam = team?.metadata?.requestedSlug ?? null;
   const isTeamAnOrg = team.isOrganization;
+  const organization = inviter.profile.organization;
+
+  let inviterOrgId = inviter.organization.id;
+  let orgSlug = organization ? organization.slug || organization.requestedSlug : null;
+  let isInviterOrgAdmin = inviter.organization.isOrgAdmin;
 
   const invitations = buildInvitationsFromInput({
-    usernameOrEmail: input.usernameOrEmail,
-    roleForAllInvitees: input.role,
+    usernameOrEmail,
+    roleForAllInvitees: role,
   });
   const isAddingNewOwner = !!invitations.find((invitation) => invitation.role === MembershipRole.OWNER);
 
   if (isTeamAnOrg) {
     await throwIfInviterCantAddOwnerToOrg();
   }
 
+  if (isPlatform) {
+    inviterOrgId = team.id;
```

Comment:
`inviter. organization` returns an empty object for platform user, and since we know every platform team is a valid team we use team here instead.